src/main/java/be/syntra/devshop/DevshopFront/services/SearchService.java

```diff
@@ -24,9 +24,15 @@
 
     void setAppliedFiltersToSearchModel();
 
-    void setPriceFilters(BigDecimal minPrice,BigDecimal maxPrice);
+    void setPriceFilters(BigDecimal minPrice, BigDecimal maxPrice);
 
     void setSortingByName();
 
     void setSortingByPrice();
+
+    void setSelectedCategory(String category);
```

Comment:
addToSelectedCategories() so we know there can be more and we're adding one

Reply:
fixed